api/envoy/extensions/formatter/metadata/v3/metadata.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package envoy.extensions.formatter.metadata.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.formatter.metadata.v3";
+option java_outer_classname = "MetadataProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Formatter extension for printing various types of metadata]
+// [#extension: envoy.formatter.metadata]
+
+// Metadata formatter extension implements METADATA command operator that
+// prints all types of metadata. The first parameter taken by METADATA operator defines
+// type of metadata. The following types of metadata are supported (case sensitive):
+//
+// * DYNAMIC
+// * CLUSTER
+// * ROUTE
+//
+// See :ref:`here <config_access_log>` for more information on access log configuration.
+
+// %METADATA(TYPE:NAMESPACE:KEY):Z%
+//    :ref:`Metadata <envoy_v3_api_msg_config.core.v3.Metadata>` info,
+//    where TYPE is type of metadata (see above for supported types),
+//    NAMESPACE is the filter namespace used when setting the metadata, KEY is an optional
+//    lookup up key in the namespace with the option of specifying nested keys separated by ':',
+//    and Z is an optional parameter denoting string truncation up to Z characters long.
+//    The data will be logged as a JSON string. For example, for the following ROUTE metadata:
+//
+//    ``com.test.my_filter: {"test_key": "foo", "test_object": {"inner_key": "bar"}}``
+//
+//    * %METADATA(ROUTE:com.test.my_filter)% will log: ``{"test_key": "foo", "test_object": {"inner_key": "bar"}}``
+//    * %METADATA(ROUTE:com.test.my_filter:test_key)% will log: ``"foo"``
```

Comment:
hmm it should be `foo` without the quotes since #16736?

Reply:
I blindly copied it from https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage.html. Corrected docs everywhere. Thanks for catching it.